CycleHireApp/app.R

```diff
@@ -0,0 +1,76 @@
+#Shiny app for cycle hire from https://github.com/Robinlovelace/geocompr/issues/584
+# Author - Kiranmayi Vadlamudi
+#25th Dec 2020
+
+library(pacman)
+p_load(sf, shiny, spData, leaflet, tidyverse, spDataLarge, units)
```

Comment:
Could you replace these lines with something that doesn't depend on pacman? It's a great package but don't want to add extra dependencies. The following lines should do the same thing so could make a good replacement:

```r
pkgs = c("sf", "shiny", "spData", "leaflet", "tidyverse", "spDataLarge", "units")
invisible(lapply(pkgs, library, character.only = TRUE))

Reply:
Done.